integration-tests/tests/src/schemas/person-and-dogs.ts

```diff
@@ -38,16 +38,13 @@ export const PersonSchema: Realm.ObjectSchema = {
 };
 
 export class Person extends Realm.Object {
```

Comment:
Isn't this supposed to extend `Realm.Object<Person>`?

Reply:
If you're not using the constructor, the type argument isn't needed.